packages/firestore/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
   small subset of the documents in a collection.
 - [fixed] Fixed a race condition between authenticating and initializing
   Firestore that could result in initial writes to the database being dropped.
+- [feature] Added a `Firestore.onSnapshotsInSync()` method that notifies users
+  when all their snapshot listeners are in sync with each other.
```

Comment:
nit: We use 2nd-person for some of the other changelog entries and I think that's probably a bit cleaner.  In particular, "users" is always a little problematic since our _users_ are the developers that are writing apps which have their own _users_.  So it can sometimes be ambiguous (not a big deal in this case probably).

> Added a `Firestore.onSnapshotsInSync()` method that notifies you when all of your snapshot listeners are in sync with each other.